bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveMultiInstanceCommandClass.java

```diff
@@ -353,34 +334,7 @@ private void handleMultiChannelCapabilityReportResponse(SerialMessage serialMess
 			}
 
 			// Add all the command classes supported by this endpoint
-			for (int i = 0; i < serialMessage.getMessagePayload().length - offset - 3; i++) {
```

Comment:
Ok - no probs - just a bit confusing (hence my stupid questions!). I’ll take a look tomorrow, but it’s probably ok from my quick look.

Thanks.